src/data/items.js

```diff
@@ -505,3 +507,25 @@ export const chocolateMilk = freeze({
   type: MILK,
   value: 80,
 })
+
+/**
+ * @property farmhand.module:items.chocolate
+ * @type {farmhand.item}
+ */
+export const chocolate = freeze({
+  id: 'chocolate',
+  name: 'Chocolate',
+  type: CHOCOLATE,
+  value: 500, // what's a good value?
+})
```

Comment:
I think that this `farmhand.item` can be removed completely, as it effectively already exists.

In short, recipes produce items that are added to the player's inventory just like a crop or a shop item. This is where that's done: https://github.com/jeremyckahn/farmhand/blob/8c8d3a1614f436e6a97a92c822fcd3b345de5077/src/data/maps.js#L24-L27

It's a little magical (AKA not entirely obvious), but this saves us from having to explicitly define recipe items twice. For every `farmhand.recipe`, there is an automatic `farmhand.item` counterpart. To see this for yourself in the `master` branch, you can make the Chocolate recipe and then run this in the console:

```js
farmhand.state.inventory.find(({ id }) => id === 'chocolate')
```

And you'll see the `chocolate` units in your inventory.

Reply:
ahhh, thank you for explaining that! i had a feeling there was something doing it since I didn't see chocolate defined in items before, but didn't figure it out. i'll clean this up now